CMakeLists.txt

```diff
@@ -2,23 +2,12 @@
 # executed from 'project/build' with 'cmake ../'.
 cmake_minimum_required(VERSION 3.9)
 project(envire_core VERSION 0.1 DESCRIPTION "Envire Graph Core Library")
-
-
-if(INSTALL_DEPS)
-  execute_process(COMMAND bash build.bash ${CMAKE_INSTALL_PREFIX}
-                  WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/source_dependencies)
-  set(ENV{PKG_CONFIG_PATH} ${CMAKE_INSTALL_PREFIX}/lib/pkgconfig/)
-  set(ENV{CMAKE_PREFIX_PATH} ${CMAKE_INSTALL_PREFIX}/share/rock/cmake/)
-endif()
-
-
```

Comment:
any reason to remove this (and the explanation from the readme)?

Reply:
None, the merge is non-trivial and i did not spot that during my review.